src/components/atoms/FaqIcon.js

```diff
@@ -0,0 +1,23 @@
+import ChatBubbleIcon from '@mui/icons-material/ChatBubble';
+import { Button } from '@mui/material';
+
+function FaqIcon() {
```

Comment:
the reason I am permiting this without props is because this atom is the same through out the app.. take note

Reply:
Noted